lib/dry/validation/hint_compiler.rb

```diff
@@ -61,6 +61,11 @@ def visit_key(node)
         name
       end
 
+      def visit_attr(node)
+        name, _ = node
```

Comment:
Think it would make more sense to just return `node.first` here and in `visit_key`